longformer/longformer.py

```diff
@@ -283,11 +279,21 @@ def forward(
 
         if getattr(self, 'global_tokens', 0) > 0:
             assert not self.use_global_proj  # TODO: support the use of global projections
-            attn_weights = torch.einsum('blhd,bshd->blhs', (q[:, :self.global_tokens], k))
+            # hidden_states shape: seqlen x batch x dim
+            q_g = self.query_global(hidden_states[:self.global_tokens])
```

Comment:
nit: I would call this `selected_q_g` just to match the naming convention used earlier.